pihole

```diff
@@ -122,6 +122,20 @@ if [[ $# = 0 ]]; then
     helpFunc
 fi
 
+#Source the setupVars from install script for the IP
+export setupVars=/etc/pihole/setupVars.conf
+
+export basename=pihole
```

Comment:
I believe exporting all of these variables and values will leave them in the environment even after the script is finished executing. I'll need to check on that however.